MhLabs.AwsSignedHttpClient/ServiceCollectionExtension.cs

```diff
@@ -28,8 +30,9 @@ public static class ServiceCollectionExtension
         private static IServiceCollection AddMhHttpClient<TClient, TImplementation>(this IServiceCollection services, HttpOptions options = null) where TClient : class
             where TImplementation : class, TClient
         {
-            if (options == null) options = new HttpOptions();
+            if (options == null) options = new HttpOptions(NullLogger.Instance);
```

Comment:
Isn't this the actual breaking change? If we change it to a default implementation for ILogger of console logger everything will work as expected incl. template literals? That is probably nicer.

Reply:
Also IMO logging has nothing to do with http options so I would prefer passing it as a separate argument to AddSignedHttpClient/AddUnsignedHttpClient/AddMhHttpClient.

Reply:
Yeah, I know. Extension methods have its limits sometimes.